forest/benchmarking/qubit_spectroscopy.py

```diff
@@ -83,7 +83,7 @@ def get_stats_by_qubit(expt_results: List[List[ExperimentResult]]) \
 # ==================================================================================================
 
 
-def generate_t1_experiments(qubits: Sequence[int], times: Sequence[float]) \
+def generate_t1_experiments(qubits: Sequence[int], times: Sequence[float], is_qvm: bool = False) \
```

Comment:
Why has the QVM stopped accepting this PRAGMA? While the more modern way of representing the delay is the Quil-T DELAY command (not a pragma), this PRAGMA should still be valid. This sort of fix by needing the user to make a test on the original quantum computer object is a bit of a non-ideal work-around, but I'm wondering if it is really necessary?

Reply:
Traced this issue back to the `quil-rs` compiler's handling of `PRAGMA` commands.

Hoping to fix this with https://github.com/rigetti/quil-rs/pull/302

Reply:
Yes, Quil-T's DELAY is the recommended way to express these, but Quil-T is not supported by the QVM. As you say, as long as Quil-T instructions are stripped prior to submission to the QVM, the right behavior should be restored and without needing this flag-based work-around.

Reply:
Changed this from preventing the instructions from being added, to pruning them right before execution, removing the need for a flag https://github.com/rigetti/forest-benchmarking/pull/237/commits/1b2b4313a3338042cd8b81f414b49dbdcd8f0d22

Reply:
Nice!